.github/workflows/pr-validations.js

```diff
@@ -0,0 +1,132 @@
+const path = require("path")
+const fs = require("fs")
+const GitHub = require("github-api")
+
+const OWNER = "aws"
+const REPO = "aws-cdk"
+
+class ValidationFailed extends Error {
+    constructor(message) {
+        super(message);
+    }    
+}
+
+function createGitHubClient() {
+
```

Comment:
Why the newline at the beginning of each function? Is that an intentional stylistic decision?

Reply:
Actually yes, I use a new line when the function body isn't a trivial one liner or something. But I am happy to change my ways if we have different conventions :)

Reply:
We never discussed this, but the de-facto style in our code base is not to have a newline at the beginning of the function. It's not that critical, so if you feel this makes the code more readable/maintainable, feel free to use this style. Just wanted to know if it was intentional.

Reply:
I also don't feel to strongly about it, so I will rather conform to our current style. Will change.